src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1975,54 +1975,79 @@ limitations under the License.
-            <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Channel" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Audio Output" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Low Power" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Wake on LAN" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Target Navigator" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Account Login" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Channel" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Audio Output" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Low Power" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Wake on LAN" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Target Navigator" client="false" server="true" clientLocked="false" serverLocked="true"></include>
+        </clusters>
+    </deviceType>
+    <deviceType>
+        <name>MA-casting-videoclient</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Casting Video Client</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0029</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Media Playback" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Content Launcher" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Keypad Input" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Account Login" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="On/Off" client="true" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Level Control" client="true" server="false" clientLocked="true" serverLocked="true"></include>
```

Comment:
Why is this serverLocked=true?  Where is this device type coming from?

Reply:
Updated per spec.